modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ConfigurationModelValidator.java

```diff
@@ -28,7 +32,9 @@
 import org.mule.runtime.extension.api.loader.Problem;
 import org.mule.runtime.extension.api.loader.ProblemsReporter;
 import org.mule.runtime.module.extension.internal.loader.java.property.ConfigTypeModelProperty;
+import org.mule.runtime.module.extension.internal.loader.java.type.property.ExtensionParameterDescriptorModelProperty;
 
+import java.lang.reflect.Modifier;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.lang.reflect.Modifier'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7618&issues=AWhxwXz1otIXnmsUr_R_&open=AWhxwXz1otIXnmsUr_R_)

[ISSUE_KEY]: AWhxwXz1otIXnmsUr_R_